Mathlib/SetTheory/Ordinal/Veblen.lean

```diff
@@ -343,6 +406,120 @@ theorem veblen_eq_veblen_iff :
 
 end veblen
 
+/-! ### Inverse Veblen function -/
+
+/-- For any given `x`, there exists a unique pair `(o, a)` such that `ω ^ x = veblen o a` and
+`a < ω ^ x`. `invVeblen₁ x` and `invVeblen₂ x` return the first and second entries of this pair,
+respectively.
```

Comment:
Can we mention the fact that `veblen_eq_opow_iff` proves this defining property to this docstring too?